cmd/prombench/clone.sh

```diff
@@ -6,5 +6,6 @@ git clone $PROMBENCH_REPO $PROMBENCH_DIR
 # copy the prombench binary to the cloned directory
 cp /usr/bin/prombench $PROMBENCH_DIR/
 
+cd $PROMBENCH_DIR
 # execute arguments passed to the image
-$@
+$*
```

Comment:
why this change?

Reply:
We tried this for github actions. But both `*` and `@` work.

Reply:
So what is the answer to the question why do we need this change?

Reply:
In github actions example, `*` was used to accept the args. So we thought that might be the case that it was not executing `make` commands. But adding `cd $PROMBENCH_DIR` resolved it. Will revert it back to `@`.